### ISSUE TITLE
Keeweb for Linux deb installs but will not launch

### ISSUE THREAD

**Issue by coffseducation** [keeweb/keeweb#2116]
**Describe the bug**
Keeweb for Linux deb installs but will not launch

**To Reproduce**
1. Install deb
2. Click on application
3. No response
4. See error from terminal below.

**Environment**
Operating System: Kubuntu 23.04
KDE Plasma Version: 5.27.7
KDE Frameworks Version: 5.109.0
Qt Version: 5.15.8
Kernel Version: 6.2.0-27-generic (64-bit)
Graphics Platform: X11
Processors: 8 × Intel® Core™ i7-7820HQ CPU @ 2.90GHz
Memory: 31.2 GiB of RAM
Graphics Processor: Quadro M1200/PCIe/SSE2
Manufacturer: Dell Inc.
Product Name: Precision 5520

KeeWeb-1.18.7

**Logs**
keeweb
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[33349:0911/105131.764945:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
/usr/bin/keeweb: line 2: 33349 Trace/breakpoint trap   (core dumped) /usr/share/keeweb-desktop/keeweb "$@"

**Comment by mspoehrle** [comment]
Hi,

I had the same problem.

It was a malfunctioning nvidia driver with hardware acceleration. When you run `vainfo` it lists the file which it tries to load but this nvidia driver file does not exist.

I change the nvidia PRIME profile to intel with the command

`sudo prime-select intel`

After reboot KeeWeb started again. 

You will loose graphic performance. I use my notebook only for some office work, so it does not matter for me.

Hope it helps

Michael

**Comment by coffseducation** [comment]
@mspoehrle Thanks Michael!
I do have Nvidia on this laptop!
I ended up using the appimage which worked.
However, I was not able to make a connection to Nextcloud.
After entering the path to the kdbx file and attempting to login I would be sent back to the login screen with no error.
I have decided to stick with Nextcloud Passwords.
nb. It is a beautiful and full featured desktop client. Someone should fork it for use with NC Passwords! 😁
